x-pack/plugins/cloud_security_posture/kibana.jsonc

```diff
@@ -21,14 +18,10 @@
       "charts",
       "discover",
       "cloud",
-      "licensing"
-    ],
-    "optionalPlugins": [
-      "usageCollection"
+      "licensing",
+      "share"
     ],
-    "requiredBundles": [
-      "kibanaReact",
-      "usageCollection"
-    ]
+    "optionalPlugins": ["usageCollection", "licenseManagement"],
+    "requiredBundles": ["kibanaReact", "usageCollection"]
```

Comment:
do we use `licenseManagement`? if not, please remove and revert the formating for the other properties